src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts

```diff
@@ -78,7 +78,8 @@ export const NavigationLinks = styled.ul<ToggleProps>`
 `;
 
 export const NavigationLinkItem = styled.li<NavigationProps>`
-  padding: 0rem 1rem;
+  padding: ${props =>
+    props.padding === 'forEmail' ? '0rem 1rem 0rem 0rem' : '0rem 1rem'};
```

Comment:
what is the purpose of passing `padding` props?

Reply:
oh, I forgot to remove those! My bad, I'll remove it asap.